assets/src/admin/paired-browsing/client.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+const { parent } = window;
+
+if ( parent.pairedBrowsingApp ) {
+	window.ampPairedBrowsingClient = true;
+	const app = parent.pairedBrowsingApp;
+
+	app.registerClientWindow( window );
+
+	domReady( () => {
+		if ( app.documentIsAmp( document ) ) {
+			// Hide the paired browsing menu item.
+			const pairedBrowsingMenuItem = document.getElementById( 'wp-admin-bar-amp-paired-browsing' );
+			if ( pairedBrowsingMenuItem ) {
+				pairedBrowsingMenuItem.remove();
+			}
+
+			// Hide menu item to view non-AMP version.
+			const ampViewBrowsingItem = document.getElementById( 'wp-admin-bar-amp-view' );
+			if ( ampViewBrowsingItem ) {
+				ampViewBrowsingItem.remove();
+			}
+		} else {
+			/**
+			 * Override the entire AMP menu item with just "Non-AMP". There should be no link to
+			 * the AMP version since it is already being shown.
+			 */
+			const ampMenuItem = document.getElementById( 'wp-admin-bar-amp' );
+			if ( ampMenuItem ) {
+				ampMenuItem.innerHTML = 'Non-AMP';
```

Comment:
Will need to be translatable.

Reply:
Still needs to be translatable. At the same time, now that we have the AMP/non-AMP message in the header, do we even need this label in the admin bar anymore? Should we just remove the AMP nav menu item altogether here?

```suggestion
				ampMenuItem.parentNode.removeChild( ampMenuItem );
```

This would prevent the user from trying to access the AMP version from the non-AMP version, which doesn't make sense since they are in paired browsing. There's no need for that admin bar menu item in the non-AMP page, now that there is the blue paired browsing bar.

Reply:
@pierlon This is still missing its "translatability". You should add the translated string on the PHP side to the obejct you're sending over to the client side via `wp_localize_script()`.